### DISCUSSION TITLE
Round Robin for moderating Friday's Open Meetings

### DISCUSSION THREAD

**Discussion by evegufy** [eclipse-tractusx/sig-infra#535] [General]
This discussion should help organise the round robing moderation of [Friday's Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings)

Based on the [discussion](https://github.com/orgs/eclipse-tractusx/projects/61/views/6?pane=issue&itemId=61108296) in the last committer meeting (July 19, 2024), I consolidated the list of regular moderators for the [**Office Hour**](https://eclipse-tractusx.github.io/community/open-meetings#Community%20Office%20Hour) to the following proposed schedule for the next couple of month:

- [x] @mgarciaLKS  2025-06-13
- [x] @dhiren-singh-007 2025-06-20
- [x] @saadanzari 2025-06-27
- [x] @tom-rm-meyer-ISST 2025-07-04
- [x] @manojava-gk  2025-07-11
- [x] @MonikaJacobsen 2025-07-18
- [x] @ds-meberle 2025-07-25
- [x] @arnoweiss 2025-08-01
- [x] @matbmoser 2025-08-08
PUBLIC HOLYDAY 2025-08-15
- [x] @stephanbcbauer 2025-08-22
- [x] @ds-hzimmer 2025-08-29
- [x] @CDiezRodriguez  2025-09-05
- [x] @ther3sa 2025-09-12
- [x] @lgblaumeiser 2025-09-19
- [x] @Phil91 2025-09-26

If you are a committer, please also take over the moderation of the [**Committer Meeting**](https://eclipse-tractusx.github.io/community/open-meetings/#Committer%20Meeting), taking place every second week Friday, just right after the Office Hour. If you'd like to be very supportive, write quick reminder about the upcoming meeting into the [Committer Matrix Chat](https://chat.eclipse.org/#/room/#tractusx-committers:matrix.eclipse.org) or on the dev Mailing List a couple of days before :)

**Comment by matbmoser** [comment]
Fine for me! 💯

**Comment by tom-rm-meyer-ISST** [comment]
Fine for me. Maybe also mention the committer office hour?

**Reply by evegufy** [reply]
thanks for the hint 👍  I adjusted the description

**Comment by ther3sa** [comment]
If you need more moderators I would also offer myself as a moderator. :)

**Reply by evegufy** [reply]
great, I added you to the schedule :)

**Comment by ds-meberle** [comment]
Fine for me!

**Comment by Phil91** [comment]
fine for me :)

**Comment by ds-hzimmer** [comment]
I'll be out of office 2024-08-02 (and 2024-08-09)

@tom-rm-meyer-ISST offered to potentially switch with him taking over next week 2024-08-02 and then I could do 2024-09-20

**Reply by evegufy** [reply]
I did the switch in the description 👍

**Comment by Siegfriedk** [comment]
Fine for me. There is a minimal risk that i have to travel to buy something. I appoligize upfront if that will happen at that week

**Reply by evegufy** [reply]
Hi @Siegfriedk changed from 2024-09-13 to 2024-10-18 due to your sick leave, get well soon!

**Comment by danielmiehle** [comment]
Fine for me!

**Comment by evegufy** [comment]
Would somebody be available to switch with the slot from @giterrific on 2024-08-30? He's not available until the end of August.

**Reply by evegufy** [reply]
@arnoweiss switched with  @giterrific on 2024-08-30.
I added @giterrific for 2024-09-27. Is that fine for you?

**Reply by giterrific** [reply]
Works for me, thx!

**Comment by lgblaumeiser** [comment]
Hi @evegufy , you can add me to the list of moderators.

**Reply by evegufy** [reply]
Hi @lgblaumeiser great! would 2024-10-18 work for you? (@Siegfriedk mentioned he won't be able to cover the slot)

**Reply by evegufy** [reply]
Hi @lgblaumeiser I added you for 2024-10-18, just let me know in case the slot doesn't fit you

**Reply by lgblaumeiser** [reply]
Hi @evegufy , Oct. 18th sounds good to me

**Comment by evegufy** [comment]
Daniel isn't available to today, Arno will take over, this results in the following switch:
@arnoweiss 2024-11-22
@danielmiehle 2024-11-29

**Comment by evegufy** [comment]
to all in the following list, could you please confirm that the slots work for you?

- [x] @stephanbcbauer 2024-12-06

- [x] @evegufy 2024-12-13

- [x] @ds-hzimmer 2024-12-20

CHRISTMAS / NEW YEAR

- [x] @Phil91 2025-01-03

- [x] @ther3sa 2025-01-10

- [ ] @giterrific 2025-01-17

- [x] @lgblaumeiser 2025-01-24

**Comment by typecastcloud** [comment]
Hi! I would also like to join the round Robin. Thanks!

**Reply by evegufy** [reply]
@typecastcloud would 2025-01-17 work for you? @giterrific won't be to cover the slot.

**Reply by typecastcloud** [reply]
Ok 👍

**Comment by manojava-gk** [comment]
@evegufy I would like to moderate it. Pls add me to the list

**Reply by evegufy** [reply]
@manojava-gk would 2025-01-31 work for you?

**Reply by manojava-gk** [reply]
ok

**Comment by dhiren-singh-007** [comment]
Hi! I am happy to join the round Robin. Please add me in the list. Thanks!

**Reply by evegufy** [reply]
@dhiren-singh-007  would 2025-02-07 work for you?

**Reply by dhiren-singh-007** [reply]
@evegufy Sorry, currently I am on vacation, so I won't be available. Can you please set any date after 17th Feb 2025?

**Reply by evegufy** [reply]
@dhiren-singh-007 np, I assigned 2025-02-21 to you.

**Comment by saadanzari** [comment]
Hi Team, I would like to join the Round Robin. Can you please include me in the list? thanks

**Reply by evegufy** [reply]
@saadanzari  would 2025-02-14 work for you?

**Reply by saadanzari** [reply]
@evegufy yes it will work for me. thanks

**Reply by evegufy** [reply]
@saadanzari the slot is assigned to you

**Comment by MonikaJacobsen** [comment]
Hi @evegufy,
I would also like to join the round robin. Please feel free to add me to the list.
Thank you

**Reply by evegufy** [reply]
@MonikaJacobsen would 2025-02-21 work for you?

**Reply by MonikaJacobsen** [reply]
Thank you for the planing. Could I do the moderation on 2025-02-28 maybe? On the 2025-02-21 I can only join the first 30 minutes :-(

**Reply by evegufy** [reply]
@MonikaJacobsen np, I assigned 2025-02-28 to you.

**Comment by danielmiehle** [comment]
Hi all, can you please remove me from the Round Rubies list - unfortunately I can no longer make it due to time constraints.

fyi @evegufy @stephanbcbauer

**Comment by evegufy** [comment]
to all in the following list, could you please confirm that the slots work for you?

- [x] @matbmoser 2025-03-07
- [x] @ds-meberle 2025-03-14
- [x] @arnoweiss 2025-03-21
- [x] @CDiezRodriguez  2025-03-28
- [x] @mgarciaLKS 2025-04-04
- [x] @ds-hzimmer 2025-04-11
- [x]  @Phil91 2025-04-25
- [x] @ther3sa 2025-05-02
- [x] @typecastcloud  2025-05-09
- [x] @lgblaumeiser  2025-05-16

**Comment by CDiezRodriguez** [comment]
Hi @evegufy,
I’d be happy to be included in the round robin. Could you add me to the list?
Thank you!

**Reply by evegufy** [reply]
@CDiezRodriguez would the slot of the 2025-03-28 work for you?

**Reply by CDiezRodriguez** [reply]
It's okay for me, thanks

**Comment by mgarciaLKS** [comment]
Hey @evegufy.
I would love to be included in the Round Robin when posible. :smile:

**Reply by evegufy** [reply]
Hi @mgarciaLKS would the slot of the 2025-04-04 work for you?

**Reply by mgarciaLKS** [reply]
perfect :grinning:

**Comment by evegufy** [comment]
@ all in the following list, could you please confirm if the slot works for you?

- [x] @mgarciaLKS  2025-06-13
- [x] @dhiren-singh-007 2025-06-20
- [x] @saadanzari 2025-06-27
- [x] @tom-rm-meyer-ISST 2025-07-04
- [x] @manojava-gk  2025-07-11
- [x] @MonikaJacobsen 2025-07-18
- [x] @ds-meberle 2025-07-25
- [x] @arnoweiss 2025-08-01
- [x] @matbmoser 2025-08-08
PUBLIC HOLYDAY 2025-08-15
- [x] @stephanbcbauer 2025-08-22
- [x] @ds-hzimmer 2025-08-29
- [x] @CDiezRodriguez  2025-09-05
- [x] @ther3sa 2025-09-12
- [x] @lgblaumeiser 2025-09-19
- [x] @Phil91 2025-09-26

**Reply by mgarciaLKS** [reply]
very good morning, I am on vacation in August, so I will not be able to.

**Reply by evegufy** [reply]
after feedback from @mgarciaLKS and @CDiezRodriguez I did switch the slots with @matbmoser and @stephanbcbauer, and I also remembered that @Phil91 is on leave in August so I change to September

**Reply by stephanbcbauer** [reply]
thx, fine for me :)

**Reply by matbmoser** [reply]
I accepted

**Reply by matbmoser** [reply]
2025.08-22 is just public holiday in bÄyern

**Reply by manojava-gk** [reply]
I'll be on leave next week, So i will not be able to moderate

**Reply by evegufy** [reply]
@manojava-gk I talked to Mikel and he'll take over 2025-06-13, could you please confirm if you can take over 2025-07-18?

**Reply by manojava-gk** [reply]
Thanks for checking  but I'm sorry, I will be on vacation again from 16th to 25th.

**Reply by evegufy** [reply]
@manojava-gk so 2025-07-11 would work? @MonikaJacobsen would you be open to change to 2025-07-18 so that Mano could cover 2025-07-11?

**Reply by manojava-gk** [reply]
Yes works

**Reply by MonikaJacobsen** [reply]
That's fine for me. I'll take over 2025-07-18

**Reply by evegufy** [reply]
@MonikaJacobsen great I edited the list

**Reply by MonikaJacobsen** [reply]
@rogocof as discussed last week, could you please take over the moderation of the meetings on Friday, July 25, 2025?

**Comment by ReneSchroederLJ** [comment]
I would like to be added to the list of possible moderators. Thank you

**Reply by matbmoser** [reply]
I wil include you in the new list

**Comment by matbmoser** [comment]
I have open this in another Discussion: https://github.com/eclipse-tractusx/sig-infra/discussions/556
